tests/main.py

```diff
@@ -20,6 +20,7 @@
 ADDITIONAL_CHECKERS = [
     checkers.resource_files.check_image_size,
     checkers.resource_files.check_resource_file_name,
+    checkers.resource_files.check_is_resource_file_used,
```

Comment:
The additional checks are only run in review mode, but this new check should be part of the standard test suite run by Runbot.